src/components/LoginContainer.js

```diff
@@ -27,26 +37,17 @@ class LoginContainer extends Component {
     }
 
     _email() {
-        return (i => {
-            this.email = i;
+        return (ref => {
+            this.email = ref;
         });
     }
 
     _password() {
-        return (i => {
-            this.password = i;
+        return (ref => {
```

Comment:
I mean, `i` stands for input sooooo.... https://facebook.github.io/react/docs/refs-and-the-dom.html